db/migrate/20220810131220_add_automated_activity_to_audits.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class AddAutomatedActivityToAudits < ActiveRecord::Migration[6.1]
+  # rubocop:disable Metrics/MethodLength
+  def up
+    add_column(
+      :audits,
+      :automated_activity,
+      :boolean,
+      default: false,
+      null: false
```

Comment:
Does it make sense to have `null: false` if the column has a default? I can't decide!